lib/code-manager.js

```diff
@@ -121,7 +121,10 @@ export function getCodeToInspect(editor: atom$TextEditor) {
 export function getRegexString(editor: atom$TextEditor) {
   const scope = editor.getRootScopeDescriptor();
 
-  const { commentStartString } = editor.getCommentStrings(scope);
+  const {
+    commentStartString
+    // $FlowFixMe: This is an unofficial API
+  } = editor.getScopedSettingsDelegate().getCommentStrings(scope);
```

Comment:
This still is a experimental API, but it's available in Atom 1.23 as well

Reply:
I think that's fine for now, we can ship patches if/when it changes. Luckily with updates once it breaks we can address the new usage and people get the fixes quick.